chainer/training/extensions/evaluator.py

```diff
@@ -1,17 +1,10 @@
-import copy
-
-import six
-
 from chainer.dataset import convert
-from chainer.dataset import iterator as iterator_module
-from chainer import link
 from chainer import reporter as reporter_module
 from chainer.training import extension
-from chainer import variable
+from chainer.training import StandardEvaluator
```

Comment:
We forbid the direct import of class objects (See the [coding guideline](http://docs.chainer.org/en/stable/contribution.html#coding-guidelines) for detail). So could you replace it with `from chainer.training import evaluator` and use `evaluator.StandardEvaluator` instead?